src/app/accounts/two-factor.component.html

```diff
@@ -100,6 +100,13 @@ <h1>{{ title }}</h1>
         </div>
       </div>
     </div>
+    <div class="box last" [hidden]="!showCaptcha()">
+      <div class="box-content">
+        <div class="box-content-row">
+          <iframe id="hcaptcha_iframe" height="80"></iframe>
```

Comment:
Won't this behave weirdly when the captcha expands to be solved?

Reply:
No sir! Magic
![Screen Shot 2022-03-03 at 4 12 09 PM](https://user-images.githubusercontent.com/26154748/156661460-26229289-70f5-4246-9076-78f2b1ae5dfb.png)
!